js-webapp-async/README.md

```diff
@@ -99,7 +99,11 @@ onmessage = function(message) {
     });
   }
 };
-```
+```
+
+The figure below illustrates what is happening in parallel in the two threads.
+
+![threads](threads.png)
```

Comment:
Could you replace `worker.postMessage()` with `w.postMessage()`? For consistency